game-headed/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
This is new, could you provide a reference for what this file is doing or quickly explain it? Does this take the place of `@ExtendWith` at all?

Reply:
Ah, this is just a file that enables mockito to be able to mock final classes, which is required to mock the webview class

Reply:
Interesting, thanks for the explanation